multichain-testing/package.json

```diff
@@ -4,9 +4,7 @@
   "type": "module",
   "scripts": {
     "build": "exit 0",
-    "lint": "yarn lint:eslint",
-    "lint:eslint": "eslint .",
-    "lint:types": "tsc --noEmit",
+    "lint": "yarn tsc && yarn eslint .",
```

Comment:
🎉 at one point, i don't think this was possible due to type errors from dependencies